### PR TITLE
added code for parsing plist data files generated by Coco Studio

### PR DESCRIPTION
**Pull request by pangliang**
The two format corresponds to one ext, so i change the parameter 'ext' to 'format'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +51,41 @@ If you have **Sprite.json** data file instead of the **Sprite.plist** one run th
     python unpacker.py Sprite json
     
 Result will be the same.
+
+### CocoStudio Plist example
+
+If you have **Sprite.plist** data file, it is defferent from the plist which packed by TexturePacker
+
+```xml
```
Comment (pavle-goloskokovic) [inline README.md]: Could you remove this xml code from README.md and maybe add folder examples/cocos/ and put in it atlas image and .plist file generated from CocoStudio. That way I can test if the code is working and users can have something to test this code with in the future. I'll be adding examples for json and plist formats, maybe we can use the same sprites for generating all atlases, let me know if you find any free nice looking ones...

Thanks!

**Comment by pangliang** [comment]
hello, i delete it

**Comment by pavle-goloskokovic** [comment]
Hey, great!

What about an example atlas image and data file generated in Coco Studio?

**Comment by pavle-goloskokovic** [comment]
@pangliang there are conflicts, could you resolve them?

**Pull request closed without merging**